src/govuk/components/header/header.yaml

```diff
@@ -51,11 +51,15 @@ params:
 - name: navigationLabel
   type: string
   required: false
-  description: Text for the `aria-label` attribute of the navigation. Defaults to 'Navigation menu'.
+  description: Text for the `aria-label` attribute of the navigation. Defaults to 'Menu'.
```

Comment:
Small one, I think the default is "Show or hide menu" instead of "Menu"

Reply:
You might be mixing `navigationLabel` with `menuButtonLabel` — the latter parameter does default to 'Show or hide menu'.

They're pretty similarly named parameters 😅